components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/http/upload.ts

```diff
@@ -17,11 +17,11 @@ import * as streams from "@dirigible/io/streams"
 import * as bytes from "@dirigible/io/bytes"
 const HttpUploadFacade = Java.type("org.eclipse.dirigible.components.api.http.HttpUploadFacade");
 
-export function isMultipartContent() {
+export function isMultipartContent(): boolean {
     return HttpUploadFacade.isMultipartContent();
 }
 
-export function parseRequest() {
+export function parseRequest(): FileItems {
```

Comment:
Where the `FileItems` type/interface is defined?

Reply:
The class "FileItems" is defined right after the method.